applications/dashboard/design/style.css

```diff
@@ -6214,15 +6198,9 @@ img.jcrop-preview {
 }
 
 .flex {
-  display: -webkit-flex;
-  display: -ms-flexbox;
   display: flex;
-  -webkit-justify-content: space-between;
```

Comment:
Looks like we lost a lot of auto prefixing? Did our settings change?

Reply:
Whoops. Fixed.